web-src/src/components/ModalDialogAlbum.vue

```diff
@@ -12,8 +12,9 @@
               <p class="title is-4">
                 <a class="has-text-link" @click="open_album">{{ album.name }}</a>
               </p>
-              <div class="buttons" v-if="media_kind === 'podcast' && new_tracks > 0">
-                <a class="button is-small" @click="mark_played">Mark as played</a>
+              <div class="buttons" v-if="media_kind === 'podcast'">
+                <a class="button is-small" v-if="new_tracks > 0" @click="mark_played">Mark as played</a>
+                <a class="button is-small" @click="$emit('remove_podcast')">Remove podcast</a>
```

Comment:
would it be possible to add `is-danger` modifier to the `Remove podcast`?

Reply:
I don't want to make the button that prominently visible. It only has a function for podcasts added over the JSON API. For podcasts from the local library this button only leads to an error message.

To avoid accidental deletion, there is now a confirmation dialog displayed before actually removing the podcast.